hw_diag/diagnostics/env_var_diagnostics.py

```diff
@@ -19,6 +19,9 @@
 }, {
     'ENV_VAR': 'VARIANT',
     'DIAGNOSTIC_KEY': 'VA'
+}, {
+    'ENV_VAR': 'FIRMWARE_SHORT_HASH',
+    'DIAGNOSTIC_KEY': 'FSH'
```

Comment:
@MuratUrsavas - Why are we adding this here? We can just display the hash in the `/version` endpoint. Don't think we need that in `/initFile.txt` payload.

Reply:
I thought it would be nice to report the hash where the FIRMWARE_VERSION is reported. I didn't know the direction and what was going to be deprecated in the near future.

Reply:
Thought again, and we have to have this duo anyway, because the mapping has to include FIRMWARE_SHORT_HASH (We're getting it from an environment variable) and in the current scenario we need to give it a short name. If we'd remove the short naming scheme, then this would go away too. I making it two chars as FH.

Reply:
Ok this gets used in the web ui too so it's fine to keep it until we start using a different method for displaying this info in the web interface.